src/github_domain.py

```diff
@@ -20,41 +20,46 @@
 
 from typing import Any, Dict, List, Optional, Type
 
-DEFAULT_TIMESTAMP: datetime.datetime = (
-    datetime.datetime(datetime.MINYEAR, 1, 1, tzinfo=datetime.timezone.utc))
-
 
 class Assignee:
     """A class representing an assignee of a pull request."""
 
     def __init__(
         self,
         username: str,
-        timestamp: datetime.datetime = DEFAULT_TIMESTAMP
+        assigned_on_timestamp: Optional[datetime.datetime] = None
     ) -> None:
         """
         Args:
             username: str. GitHub username of the assignee.
-            timestamp: datetime. The time when an event happened.
+            assigned_on_timestamp: datetime|None. The time when the reviewer was
+                assigned, else None.
         """
         self.username = username
-        self.timestamp = timestamp
+        self.assigned_on_timestamp = assigned_on_timestamp
 
-    def set_timestamp(self, timestamp: datetime.datetime) -> None:
+    def set_assigned_on_timestamp(self, assigned_on_timestamp: datetime.datetime) -> None:
         """Sets timestamp to assignee.
+
         Args:
-            timestamp: datetime. the time when an event happened.
+            assigned_on_timestamp: datetime. The time when the reviewer was
+                assigned to the PR.
         """
-        self.timestamp = timestamp
+        self.assigned_on_timestamp = assigned_on_timestamp
 
-    def get_readable_waiting_time(self) -> str:
-        """Returns readable waiting time on review.
-        waiting time = ( current time - when reviewer was assigned )
+    def get_waiting_time(self) -> str:
+        """Returns the waiting time on review.
+        The waiting time is the difference between the current time and the time when the
+        reviewer was assigned to the PR.
 
         Returns:
-            str. the waiting time in days and hours.
+            str. The waiting time in days and hours.
         """
-        delta = datetime.datetime.now(datetime.timezone.utc) - self.timestamp
+
+        # Since a reviewer was assigned, we are not expecting the respective
+        # timestamp(when the the reviewer was assigned) to be none.
```

Comment:
...timestamp (when ... (add space before '(')

the the --> the

Reply:
Just wanted to know, are you using any extension to catch this?

Reply:
No, I'm just reading the PR "manually".

(I wanted to say "by hand" but I guess it's "by eye"? :P )

Reply:
done